src/DotLiquid/Context.cs

```diff
@@ -685,7 +694,7 @@ private static object Liquidize(object obj)
 
             if (obj.GetType().GetTypeInfo().GetCustomAttributes(typeof(LiquidTypeAttribute), false).Any())
             {
-                var attr = (LiquidTypeAttribute) obj.GetType().GetTypeInfo().GetCustomAttributes(typeof(LiquidTypeAttribute), false).First();
+                var attr = (LiquidTypeAttribute)obj.GetType().GetTypeInfo().GetCustomAttributes(typeof(LiquidTypeAttribute), false).First();

```

Comment:
Probably faster (perf) to use `FirstOrDefault` and check for null instead of call `GetCustomAttributes()` twice. Even with the internal caching the resulting array is recreated.

Reply:
I didn't change this line, it was cleaned up by the code formatter.